lizmap/modules/lizmap/classes/qgisVectorLayer.class.php

```diff
@@ -1522,8 +1534,7 @@ protected function requestPolygonFilter($editing_context = false, $ttl = 60, $ge
                 );
 
                 if ($use_cache) {
-                    // Todo: use the project cache tools
-                    // jCache::set($cache_key, json_encode($polygon_filter_data), $ttl);
+                    $cached = $this->project->getCacheHandler()->setProjectRelatedDataCache($cache_key, $polygon_filter_data, 3600);
```

Comment:
cacheHandler may be unavailable ?

catch Exception  ?

 ```
       try {
            $cached = $this->project->getCacheHandler()->setProjectRelatedDataCache($cache_key, $polygon_filter_data, 3600);
        } catch (\Exception $e) {
            // if qgisprojects profile does not exist, or if there is an
            // other error about the cache, let's log it
            \jLog::logEx($e, 'error');
        }
```

or set $use_cache=false above (when getProjectRelatedDataCache() throws Exception ?

Reply:
Keep it before with `getProjectRelatedDataCache`